packages/teams-js/src/public/video.ts

```diff
@@ -88,11 +88,33 @@ export namespace video {
     notifyError: (errorMessage: string) => void,
   ) => void;
 
+  /**
+   * Predefined failure reasons for preparing the selected video effect
+   * @beta
+   */
+  export enum EffectFailureReason {
+    /**
+     * A wrong effect id is provided
+     */
+    InvalidEffectId = 'InvalidEffectId',
```

Comment:
>InvalidEffectId = 'InvalidEffectId', [](http://example.com/codeflow?start=4&length=36)

Under what circumstances would this happen?

Reply:
It may be worth documenting a few of these here, otherwise developers may wonder under what circumstances it would be used.